grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -562,6 +566,26 @@ public GrpcServiceBuilder useClientTimeoutHeader(boolean useClientTimeoutHeader)
         return this;
     }
 
+    /**
+     * Adds a {@link GrpcHealthCheckService} to this {@link GrpcServiceBuilder}.
+     */
+    public GrpcServiceBuilder addGrpcHealthCheckService(GrpcHealthCheckService grpcHealthCheckService) {
+        requireNonNull(grpcHealthCheckService, "grpcHealthCheckService");
+        this.grpcHealthCheckService = grpcHealthCheckService;
+        return this;
+    }
+
+    /**
+     * Sets a {@link GrpcHealthCheckService} to this {@link GrpcServiceBuilder}.
+     * The gPRC health check service manages only the health checker that determines
+     * the healthiness of the {@link Server}.
+     */
+    public GrpcServiceBuilder enableGrpcHealthCheckService() {
```

Comment:
For future-proof.
```suggestion
    public GrpcServiceBuilder enableGrpcHealthCheckService(boolean enableHealthCheckService) {
```

Reply:
Could we add `<a href="https://github.com/grpc/grpc/blob/master/doc/health-checking.md">GRPC Health Checking Protocol</a>` to the Javadoc?